repaygithub/cactus
# CACTUS-783 :: Icon Alignment

Ticket: https://repayonline.atlassian.net/browse/CACTUS-783

Like I said in standup, I didn't find any regressions to fix after updating the icons. If you find any or already know of any off the top of your head, fill me in and I'll get them updated.

Looking at the storyshot diffs is probably the best way to review this; there are a lot of changes to the svg files that are more or less pointless to review in the diff.

I also fixed a little issue with the side bar in the docs website. There's been some whitespace there forever that's always bugged me, but it was a super easy fix.

### Testing
- Run the `cactus-icons` storybook and check out all of the icons. Make sure nothing sticks out.
- Run the `cactus-web` storybook and check out some components that use icons. Some good ones to check would probably be Accordion with Outline, DateInput, Select, SplitButton, MenuButton, ColorPicker, IconButton and so on.
